src/views/honeysql/util.clj

```diff
@@ -70,6 +75,11 @@
   [query op]
   (mapcat query-tables (op query)))
 
+(defn exists-tables
+  [query]
+  (when (:exists query)
```

Comment:
should the `exists` operator be also supported when nested in `and`, `or` and `not` operators?

Reply:
nvm `collect-map` is taking care of that part